### PR TITLE
Adding a logo to README.md

### LINKED ISSUE #66: Add a logo to readme file
Add a logo to readme file

### PR DESCRIPTION
**Pull request by Shalevro2**
Adding a logo to README.md

Resolve #66

### CODE REVIEW

**Review by rawadabu — APPROVED** [review]

**Review by yaelnoyman1 — APPROVED** [review]

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by finziamit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+<p align="center">
```
Comment (Yarboa) [inline README.md:2]: Why did you choose to work with html and not pure markdown?

Smarticle{ width="800" height="600" style="display: block; margin: 0 auto" }

Comment (Shalevro2) [reply README.md:2]: Acutely, I tried it before, this lines doesn't work for some reason.
I tried another way, to add a CSS file, but In my opinion it is too much to create a new file just centering the image.

I searched examples to insert an image to markdown file, and most of them use html parts, like [here](https://github.com/ElectricRCAircraftGuy/eRCaGuy_hello_world/blob/master/markdown/github_readme_center_and_align_images.md).

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**